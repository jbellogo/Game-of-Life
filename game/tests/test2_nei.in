new 5 
init
2 2
4 4
-1 -1
init
0 0
0 4
-1 -1
init
4 0
4 1
4 2
4 3
4 4
-1 -1

print
step 
print
step
print 
step 2
print

new 5
init
2 2
4 4
0 0
0 4
4 0
4 1
4 2
4 3
-1 -1
print
step 4
print
