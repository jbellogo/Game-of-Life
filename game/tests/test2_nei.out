X___X
____X
__X_X
____X
X___X
_____
____X
____X
____X
_____
_____
_____
___XX
_____
_____
_____
_____
_____
_____
_____
X___X
____X
__X_X
____X
X___X
_____
____X
____X
____X
_____
